pallets/author-inherent/src/lib.rs

```diff
@@ -83,6 +83,11 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type Author<T: Config> = StorageValue<_, T::AccountId, OptionQuery>;
 
+	/// The highest slot that has been seen in the history of this chain.
+	/// This is a strictly-increasing value.
+	#[pallet::storage]
+	pub type HighestSlotSeen<T: Config> = StorageValue<_, u32, ValueQuery>;
```

Comment:
nits:
```suggestion
	type HighestSlot<T: Config> = StorageValue<_, u32, ValueQuery>;
```
I prefer this name and keeping the `seen in the history of this chain` in the comment.

Also it doesn't need to be public?

Reply:
never mind, unnecessary tbh